RecoParticleFlow/PFClusterProducer/plugins/alpaka/PFMultiDepthClusterizer_Alpaka.h

```diff
@@ -0,0 +1,43 @@
+#ifndef PFClusterProducer_plugins_alpaka_PFMultiDepthClusterizer_Alpaka_h
+#define PFClusterProducer_plugins_alpaka_PFMultiDepthClusterizer_Alpaka_h
+
+// Alpaka includes
+#include <alpaka/alpaka.hpp>
+
+#include "DataFormats/ParticleFlowReco/interface/alpaka/PFRecHitDeviceCollection.h"
+
+#include "DataFormats/ParticleFlowReco/interface/alpaka/PFClusterDeviceCollection.h"
+#include "DataFormats/ParticleFlowReco/interface/alpaka/PFRecHitDeviceCollection.h"
+#include "DataFormats/ParticleFlowReco/interface/alpaka/PFRecHitFractionDeviceCollection.h"
+
+#include "RecoParticleFlow/PFClusterProducer/interface/alpaka/PFMultiDepthClusteringCCLabelsDeviceCollection.h"
+#include "RecoParticleFlow/PFClusterProducer/interface/alpaka/PFMultiDepthClusteringVarsDeviceCollection.h"
+#include "RecoParticleFlow/PFClusterProducer/interface/alpaka/PFMultiDepthClusteringEdgeVarsDeviceCollection.h"
+
+#include "HeterogeneousCore/AlpakaMath/interface/deltaPhi.h"
+
+#include "RecoParticleFlow/PFClusterProducer/plugins/alpaka/PFMultiDepthClusterParams.h"
+
+/**
+ * @class PFMultiDepthClusterizer_Alpaka
+ * @brief Alpaka clusterizer algorithm for multi-depth particle flow clusters.
+ * 
+ * This class manages the execution of the full multi-stage particle flow clustering pipeline
+ * using Alpaka, including link building, adjacency graph construction,
+ * connected component detection (ECL-CC), and postprocessing.
+ */
+
+namespace ALPAKA_ACCELERATOR_NAMESPACE::eclcc {
```

Comment:
Given the use of `ALPAKA_ACCELERATOR_NAMESPACE`, it would be good to
```cpp
#include "HeterogeneousCore/AlpakaInterface/interface/config.h`
```
explicitly.